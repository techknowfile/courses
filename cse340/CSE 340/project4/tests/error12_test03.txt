TYPE
    s, r, t : u;
    u, v, w : INT;
{
    a = a * 1;
}

