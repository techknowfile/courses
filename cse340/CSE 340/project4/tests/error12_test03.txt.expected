ERROR CODE 1.2 u
